lib/messageFormat/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ *  @fileOverview
+ *  messageFormat provides utility functions to which use the INTL API which provides 
+ *  language sensitive string comparison, number formatting, and date and time formatting.
+ * 
+ *  messageFormat can have functions which are used in multiple places
+ *
+ *  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
+ */
+
+
+var global = require('../global');
```

Comment:
Use `const` and `let` instead of `var` throughout the file
We don't need to follow the previously followed conventions unless the build fails

Reply:
The transpiler does not support let and const

Reply:
@RahulSubramaniam17 We don't have support of es6? If yes then we need to think about to upgrade it may be in tech debt @soumen-saniel